mongodbatlas/resource_mongodbatlas_alert_configuration.go

```diff
@@ -593,7 +593,7 @@ func flattenAlertConfigurationNotifications(notifications []matlas.Notification)
 		nts[i] = map[string]interface{}{
 			"api_token":              notifications[i].APIToken,
 			"channel_name":           notifications[i].ChannelName,
-			"datadog_api_key":        notifications[i].DatadogRegion,
+			"datadog_api_key":        notifications[i].DatadogAPIKey,
```

Comment:
Do the core issue is this typo then?  @coderGo93  are we sure this addresses the date issue noted by Angel in  
INTMDB-142 or is the data issue a side effect of this typo?

Reply:
Yeah it seems like that, I tested at first and it gave me an error like in ticket, I changed the name and it passed the test and doesn't show the error about inconsistency

Reply:
Related to date, I didn't get that warning issue about that, but created and updated parameters are setting in read so it shouldn't be a issue.

Reply:
Are you sure, the original ticket description from the testing the customer did reported that API key was showing a changed value.  I want to be 100% this is fixed for them.

Reply:
Ah, responded before I saw the first comment.  Okay, it sounds like the error was a side effect of the typo then.   I'll approve if the test to set and update passed.